src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,32 @@
-export const PostInfo = () => <>Put the post here</>;
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => {
+  const { comments, body, user, title } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+
+        <p>
+          <span> Posted by </span>
+
+          <UserInfo user={user} />
```

Comment:
user can be undefined, so let's add the check to render the component only when a user exists
```suggestion
          {user && <UserInfo user={user} />}
```